ContactDetailsApi/V1/Domain/Enums.cs

```diff
@@ -27,6 +27,17 @@ public enum SubType
         home,
         work,
         other,
-        landline
+        landline,
```

Comment:
I thought the idea was to keep to the way it is and just make some frontend changes and change the phone number description to something like 'Relationship to resident' and have that as a mandatory free text field. I personally feel as though having a drop-down for this would be difficult as there are so many options on who the number could belong to.

Reply:
I mentioned the MMH phone number descriptions and we decided to no longer display those. Before merging I will mention your comment to the team to see what they think

Reply:
Yes please keep me updated on it. Thank you

Reply:
Will do, I'll get my other PRs ready but I won't be merging any unless we're all happy with this approach

Reply:
@humulla We discussed this earlier in standup and decided to keep this approach using the dropdown.
We think that even making the description field required does not stop people from typing "N/A" or a non-descriptive-enough description. Using the dropdown will ensure that a descriptive value is picked (unless, for whatever reason, they pick "Other")

This is how it would look (heading to be changed to "Contact type":
![image](https://github.com/LBHackney-IT/contact-details-api/assets/70756861/1187a078-dd0a-4c1c-aa25-faa7b5077cfa)

Reply:
Okay, that makes sense, One question, when a user clicks 'other' do they have a text field they can provide the details in? Maybe we can do that when 'other' is chosen then the description is mandatory and has a validation that they can't enter N/A in the description. 

Also, we should add 'Sibling' and 'Resident' to the list. 

Lastly, I wonder if it would be simpler to have 'Child' instead of 'Son' and 'Daughter' and only 'Spouse' not 'Husband' and 'Wife '.

Reply:
Concur with adding Sibling to the list. Resident would be "Main number" no? So either replace this or leave it as-is.
Agree with child instead of son/daughter
Sadly, I think we'd need to keep Husband and Wife separate, thinking about the additional context of Cautionary Contacts and domestic abuse cases.

Reply:
Main number for resident should be fine - didn't see that there before